servicecontrol/contracts.md

```diff
@@ -42,10 +42,16 @@ To subscribe to the `MessageFailed` event:
 
  * Create an [NServiceBus endpoint](/nservicebus/hosting/nservicebus-host/).
  * Install the [ServiceControl.Contracts NuGet package](https://www.nuget.org/packages/ServiceControl.Contracts/).
- * Add the message mapping in the `UnicastBusConfig` section of the endpoint's app.config so that this endpoint will subscribe to the events from ServiceControl as shown:
+ * For transports using the [message-driven publish-subscribe](/nservicebus/messaging/publish-subscribe/) mechanism, either
```

Comment:
This entire section doesn't make sense to be an unordered list, and adding conditional items seems awkward.

This is a set of steps to follow, in order. I think this section should be restructured, which will allow the new parts to fit in better.

Reply:
What is exactly awkward about it? I've made those new items subitems for a good flow. How would you propose it to be changed? Even we create an order list with numbers we would have a conditional in it.

Reply:
Overall I don't feel like using bullet points at all is the right way to organize this. I'm not sure I'd want to just see an ordered list either, though. I feel like it would be better structured as more of a prose section with paragraphs, which could handle the different conditional paths better.

However, that's just my opinion, perhaps others are fine with it as-is.

Reply:
I thought about it and was thinking that it would not be enough content for sections. Happy to give it a try tough. Sometimes when you see something instead of just visualizing it in your mind it makes things clearer. I send in a proposal tomorrow